Tasks/DotNetCoreCLIV2/dotnetcore.ts

```diff
@@ -131,11 +127,9 @@ export class dotNetExe {
             }
         }
         if (failedProjects.length > 0) {
-            if (this.command === 'publish' && !MessagePrinted) {
-                tl.warning(tl.loc('NetCore3Update'));
-                MessagePrinted = true;
+            if (this.command === 'build' || this.command === 'publish' || this.command === 'run') {
```

Comment:
should this include 'pack' command as well?

Reply:
Ah it is handled separately.

Reply:
`pack` has a separate executor method, it is not part of `basicCommand`